test/functional/forall/test-forall-functors.hpp

```diff
@@ -61,6 +61,44 @@ class RangeSegmentOffsetViewTestFunctor
    VT work_view;
 };
 
+template <typename T, typename VT>
+class RangeSegment2DViewTestFunctor
+{
+public:
+   RangeSegment2DViewTestFunctor(VT& v, T n)
+   : work_view(v), N(n) { ; }
+
+   RAJA_HOST_DEVICE
+   void operator() (T idx)
```

Comment:
Do we still need these functors, now that we have turned off recursive inlining?

Reply:
I'm going to leave them in for now. I'll work on another branch with them changed back to lambda expressions and see how long that takes on travis.